libs/core/src/lib/breadcrumb/breadcrumb-item.directive.ts

```diff
@@ -45,4 +76,35 @@ export class BreadcrumbItemDirective implements AfterContentInit {
             this.routerLink = this.breadcrumbLink.routerLink;
         }
     }
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._arrowNavigation = this._breadcrumbComponent.arrowNavigation;
+    }
+
+    @HostListener('keydown', ['$event'])
+    handleKeydown(event: KeyboardEvent): void {
+        if (this._arrowNavigation) {
+            const nextElement: HTMLElement = this.elementRef.nativeElement.nextElementSibling.children[0];
+            const isNextElementLink = nextElement && nextElement.hasAttribute('href');
```

Comment:
As for me nextElementSibling.children looks not flexible enough.

Maybe it's better to listen for this event from breadcrumb component and move focus to this item if needed?
The item itself just has to implement .focus() method.
Moreover breadcrumb can rely on [FocusKeyManger service](https://material.angular.io/cdk/a11y/overview#focuskeymanager)